### ISSUE TITLE
[BUG] Boot entry show "welcome arch linux"

### ISSUE THREAD

**Issue by F0xedb** [ODEX-TOS/tos-live#4]
**Describe the bug**
Since TOS is a arch based distribution it obviously uses components like arch.
And since this distribution is pretty simular like arch "unlike manjaro which does a lot of different things" it includes a lot of arch components. However when booting the boot entrie briefly show "version" Arch linux. Where version obviously is the current version of your os. However this boot entry should be changed to TOS linux.

**To Reproduce**
1. Download the latest tos iso image (either server or client)
2. Boot into it either with real hardware or a VM.
3 Watch boot entrie

**Expected behavior**
Arch linux should be replaced with Tos linux during boot

**Comment by F0xedb** [comment]
I have checked this issue and it is now fixed. However I discovered another simular issue during luks decryption. When asked to enter the password of your partition it says something very simular and also includes arch. This is due to us building our kernel ontop of the arch kernel. And in there code (in the encryption) it says the arch version. This can easily be done by forking there kernel and altering the code. But that is not a priority for us right now. We might come back to this later
